cypress/support/accessibility/a11y-utils.js

```diff
@@ -41,78 +41,18 @@ export default (from, end) => {
-      !prepareUrl[0].startsWith("button-minor") &&
-      !prepareUrl[0].startsWith("batch-selection") &&
-      !prepareUrl[0].startsWith("carousel") &&
-      !prepareUrl[0].startsWith("badge") &&
-      !prepareUrl[0].startsWith("advanced-color-picker") &&
-      !prepareUrl[0].startsWith("preview") &&
-      !prepareUrl[0].startsWith("detail") &&
-      !prepareUrl[0].startsWith("help") &&
-      !prepareUrl[0].startsWith("heading") &&
-      !prepareUrl[0].startsWith("toast") &&
-      !prepareUrl[0].startsWith("sidebar") &&
-      !prepareUrl[0].startsWith("search") &&
-      !prepareUrl[0].startsWith("switch") &&
-      !prepareUrl[0].startsWith("dialog-full-screen") &&
-      !prepareUrl[0].startsWith("verticalmenu") &&
-      !prepareUrl[0].startsWith("message") &&
-      !prepareUrl[0].startsWith("card") &&
-      !prepareUrl[0].startsWith("date-input") &&
-      !prepareUrl[0].startsWith("step-sequence") &&
-      !prepareUrl[0].startsWith("button-toggle") &&
-      !prepareUrl[0].startsWith("profile") &&
-      !prepareUrl[0].startsWith("date-range") &&
-      !prepareUrl[0].startsWith("pages") &&
-      !prepareUrl[0].startsWith("pod") &&
-      !prepareUrl[0].startsWith("button-toggle-group") &&
-      !prepareUrl[0].startsWith("progress-tracker") &&
-      !prepareUrl[0].startsWith("portrait") &&
-      !prepareUrl[0].startsWith("draggable") &&
-      !prepareUrl[0].startsWith("definition-list") &&
-      !prepareUrl[0].startsWith("decimal") &&
-      !prepareUrl[0].startsWith("box") &&
-      !prepareUrl[0].startsWith("carbon-provider") &&
-      !prepareUrl[0].startsWith("pill") &&
-      !prepareUrl[0].startsWith("checkbox") &&
-      !prepareUrl[0].startsWith("note") &&
-      !prepareUrl[0].startsWith("textbox") &&
-      !prepareUrl[0].startsWith("textarea") &&
-      !prepareUrl[0].startsWith("dismissible-box") &&
-      !prepareUrl[0].startsWith("dialog") &&
-      !prepareUrl[0].startsWith("number-input") &&
-      !prepareUrl[0].startsWith("button") &&
-      !prepareUrl[0].startsWith("icon") &&
-      !prepareUrl[0].startsWith("pager") &&
-      !prepareUrl[0].startsWith("fieldset") &&
-      !prepareUrl[0].startsWith("form") &&
-      !prepareUrl[0].startsWith("drawer") &&
-      !prepareUrl[0].startsWith("group-character") &&
-      !prepareUrl[0].startsWith("duelling-picklist") &&
-      !prepareUrl[0].startsWith("multi-action-button") &&
-      !prepareUrl[0].startsWith("settings-row") &&
-      !prepareUrl[0].startsWith("numeral-date") &&
-      !prepareUrl[0].startsWith("global-header") &&
-      !prepareUrl[0].startsWith("grid") &&
-      !prepareUrl[0].startsWith("image") &&
-      !prepareUrl[0].startsWith("navigation-bar") &&
-      !prepareUrl[0].startsWith("popover-container") &&
-      !prepareUrl[0].startsWith("menu") &&
+      (prepareUrl[0].startsWith("flat-table") ||
+        prepareUrl[0].startsWith("hr") ||
+        prepareUrl[0].startsWith("inlile-inputs") ||
```

Comment:
@DlgSHi Could you please remove "inline-inputs" as well from here as this component has been merged already.